app.js

```diff
@@ -141,69 +277,102 @@ function checkEnemyRocketCollision(enemyX, enemyY) {
 /**
  * Bind all game keyboard controls
  */
+function handleClick (event) {
+	gutling.x = event.pageX;
+	gutling.y = event.pageY;
+  gutling.ammo -= 10;
```

Comment:
Please create a function 
```
function doSomethingWithGatling(event) {
	gutling.x = event.pageX;
	gutling.y = event.pageY;
       gutling.ammo -= 10;
  }
  ```
  
  So we will have
  ```
  function handleClick (event) {
     doSomethingWithGutling(event);
  }
 ```

This way the code will be a little bit more readable and obvious.